src/ml_gym/frontend/dashboard/src/App.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from 'react';
+import logo from './logo.svg';
+import './App.css';
```

Comment:
Codacy has a fix for the issue: [Replace `'./App.css'` with `"./App.css"`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
import "./App.css";
```